src/js/profile/chart.js

```diff
@@ -407,4 +407,87 @@ export class Chart extends Observable {
         this.vegaView.signal('applyFilter', false).run()
       }
   };
+
+  getExportData = (isArray = false) => {
+    const data = this.vegaView.data('data_formatted');
+    const primaryGroup = this.vegaView._signals['mainGroup'].value;
```

Comment:
you should get the signals this way: 

```suggestion
    const primaryGroup = this.vegaView.signal("mainGroup")
```